pyaction/PYACTION_GRUPTREE_INSERT_KW.DATA

```diff
@@ -0,0 +1,600 @@
+--
+-- PROPS SECTION
+--
+-- =================================================================================================================================
+PROPS
+--
+--       SWITCH OFF ECHOING OF INPUT FILES
+--
+NOECHO
+--
+--       LOAD PROPS SECTION INCLUDE FILES
+--
+INCLUDE
+         '$MODEL2/include/norne_pvt.inc'                   /
+INCLUDE
+         '$MODEL2/include/rock.inc'                        /
+INCLUDE
+         '$MODEL2/include/relperm_ow.inc'                  /
+INCLUDE
+         '$MODEL2/include/relperm_go.inc'                  /
+
+-- =================================================================================================================================
+--
+-- REGIONS SECTION
+--
+-- =================================================================================================================================
+REGIONS
+--
+--       LOAD REGION SECTION INCLUDE FILES
+--
+INCLUDE
+         '$MODEL2/include/eqlnum.grdecl'                   /
+INCLUDE
+         '$MODEL2/include/fipnum.grdecl'                   /
+INCLUDE
+         '$MODEL2/include/satnum.grdecl'                   /
+
+-- =================================================================================================================================
+--
+-- SOLUTION SECTION
+--
+-- =================================================================================================================================
+SOLUTION
+--
+--       DATUM   DATUM   OWC     PCOW   GOC    PCGO   RS   RV   N
+--       DEPTH   PRESS   DEPTH   ----   DEPTH  ----   OPT  OPT  OPT
+EQUIL
+         2561.59 268.55  2645.21 0.00  2561.59 0.00   1    0    0              /
+         2584.20 268.71  2645.21 0.00  2584.20 0.00   5    0    0              /
+--
+--       DEPTH    RS
+--                MSCF/STB
+--       ------   --------
+RSVD
+         2561.59   122.30
+         2597.00   110.00
+         2660.70   106.77
+         2697.00   106.77                                  / RS VS DEPTH EQUIL REGN 01
+--       ------   --------
+         2584.20   122.41
+         2599.90   110.00
+         2663.60   106.77
+         2699.90   106.77                                  / RS VS DEPTH EQUIL REGN 02
+
+--
+--       EQLNUM  EQLNUM  THPRES
+--       FROM    TO      VALUE
+THPRES
+         1       2                                         / REGN 1 TO REGN 2
+/
+--
+--       RESTART CONTROL BASIC = 4 (ALL=2, YEARLY=4, MONTHLY=5, TSTEP=6)
+--
+RPTRST
+         'BASIC=2'   'PBPD'                                /
+
+-- =================================================================================================================================
+--
+-- SUMMARY SECTION
+--
+-- =================================================================================================================================
+SUMMARY
+--
+--       ACTIVATE COLUMNAR SUMMARY DATA REPORTING OPTION
+--
+RUNSUM
```

Comment:
If this deck is ultimately going to end up in our regression test suite it is usually better to omit the `RUNSUM` (and the `SEPARATE`) keyword (i.e., to not generate the `.RSM` file).  We have had a number of issues with the `.RSM` file in the past, usually based in slightly different levels of accuracy between the `.RSM` and the `.UNSMRY` file.